src/lib/remark-plugins/rewrite-tutorial-links/utils/get-tutorial-map.ts

```diff
@@ -0,0 +1,20 @@
+import fetch from 'node-fetch'
+
+export async function getTutorialMap() {
+  let result = {}
+  const baseUrl = process.env.VERCEL_URL
```

Comment:
Is there a different / better way to get this base url?  In looking at the docs, `VERCEL_URL` seemed like the best fit.

Reply:
Does a path-only request work?

```ts
await fetch('/api/tutorials-map')
```

EDIT: Oh probably not as this runs during build too, right?

Reply:
It didn't work for me, I was getting an absolute url error 🤷 when trying to run this locally.